frontend/integration-tests/tests/monitoring.scenario.ts

```diff
@@ -105,9 +106,10 @@ describe('Monitoring: Silences', () => {
   });
 
   it('displays the Silences list page', async() => {
-    await sidenavView.clickNavLink(['Monitoring', 'Silences']);
+    await sidenavView.clickNavLink(['Monitoring', 'Alerting']);
     await crudView.isLoaded();
-    expect(monitoringView.listPageHeading.getText()).toContain('Silences');
+    await horizontalnavView.clickHorizontalTab('Silences');
+    expect($('.co-help-text').getText()).toContain('Silences temporarily mute alerts based on a set of conditions');
```

Comment:
Add something like `export const help = $('.co-help-text')` to `monitoring.view.ts` instead of using `$` here.

Reply:
Ok, I was going by 'if it is used twice, put it in a `.view.ts`, if only used once, no need to define a const, but I can add it.

Reply:
fixed